doozer/doozerlib/cli/release_gen_assembly.py

```diff
@@ -653,7 +653,7 @@ def _get_member_overrides(self):
 
         for package_name in self.force_is:
             if package_name in self.component_image_builds:
-                build_inspector: BrewBuildImageInspector = self.component_image_builds[package_name]
+                build_inspector: BrewBuildRecordInspector = self.component_image_builds[package_name]
```

Comment:
```suggestion
                build_inspector: BuildRecordInspector = self.component_image_builds[package_name]
```

Reply:
since this is generic?

Reply:
Yes it can work. But I deliberately ignored anything outside of the build-sync scope. We'll get here once we tackle  gen-assembly for Konflux